Docs update (#87)
unstoppabledomains/resolution: README.md

brad.zil resolves to zil1yu5u4hegy9v3xgluweg4en54zm8f8auwxu0xxj
```

### Dweb resolution

> Resolution library can help you out with resolving dweb records such as ipfs hash for websites or gundb username for 
> [p2p chat system](https://unstoppabledomains.com/chat)

Reviewer: ```suggestion
Resolution library can help you out with resolving DWeb records, such as IPFS hashes for websites or GunDB usernames for 
[p2p chat system](https://unstoppabledomains.com/chat).
```

```javascript
const {default: Resolution} = require('@unstoppabledomains/resolution')
const resolution = new Resolution()

function resolveIpfsHash(domain) {
  resolution.ipfsHash(domain)

>     .then(hash => console.log(`you can view website via public ipfs gateway https://gateway.ipfs.io/ipfs/${hash}`))

Reviewer: ```suggestion
    .then(hash => console.log(`You can access the website via a public IPFS gateway: https://gateway.ipfs.io/ipfs/${hash}`))
```

    .catch(console.error)
}

function resolveGunDbRecords(domain) {
  resolution.chatId(domain)

>     .then(id => console.log(`domain ${domain} has gundb chatId ${id}`))

Reviewer: ```suggestion
    .then(id => console.log(`Domain ${domain} has a GunDB chat ID: ${id}`))
```

    .catch(console.error)
  resolution.chatPk(domain)

>     .then(pk => console.log(`domain ${domain} has gunDB public key ${pk}`))

Reviewer: ```suggestion
    .then(pk => console.log(`Domain ${domain} has a GunDB public key: ${pk}`))
```

You can find all of the options for resolution cli within -h, --help flag. 

> Flag -m can retrive all metadata so you don't need to query them separately if you just want an overview of domain records

Reviewer: ```suggestion
Flag -m can retrive all metadata so you don't need to query them separately if you just want an overview of domain records.

```
